### PR TITLE
eslint => standard

### PR DESCRIPTION
**Pull request by ocke**
Split up the work started in https://github.com/zendesk/zendesk_app_framework_sdk/pull/71

This has no code changes, but a lot of linter code changes. Syntax was changed, never any code meaning.

standard

**review by commit!**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,46 +1,54 @@
...
+  loc = loc || window.location
+  var queryParams = Utils.queryParameters(loc.search)
+
+  var hashParams = Utils.queryParameters(loc.hash)
+
+  var origin = queryParams.origin || hashParams.origin
+
+  var appGuid = queryParams.app_guid || hashParams.app_guid
+
+  var client
```
Comment (danielbreves) [inline lib/index.js]: Why so many line breaks?

File: lib/utils.js [hunk 2]
```diff
@@ -1,53 +1,57 @@
...
-      value;
+  var result = {}
+
+  var keyValuePairs
+
+  var keyAndValue
+
+  var key
+
+  var value
```
Comment (danielbreves) [inline lib/utils.js]: Why so many line breaks?

File: lib/index.js (hunk 1 above)
Comment (ocke) [reply lib/index.js]: yeah i know, i have no idea why standard did that! This was after `standard --fix` 🤷‍♂️

Comment (ocke) [reply lib/index.js]: I removed these linebreaks.

File: lib/utils.js (hunk 2 above)
Comment (ocke) [reply lib/utils.js]: I removed the added linebreaks.

**Comment by ocke** [comment]
rebased on master.

**Pull request merged**